host-linux/Signer.h

```diff
@@ -210,7 +210,7 @@ class Signer: public QDialog {
             pin->setValidator(new QRegExpValidator(QRegExp(QString("\\d{%1,12}").arg(minPinLen)), pin));
             pin->setMaxLength(12);
             connect(pin, &QLineEdit::textEdited, [=](const QString &text){
-                ok->setEnabled(text.size() >= minPinLen);
+                ok->setEnabled((unsigned long)text.size() >= minPinLen);
```

Comment:
This gives "warning use of old-style cast"
Maybe: 
ok->setEnabled(text.size() >= int(minPinLen))

Reply:
Indeed, that works fine here (GCC 7.2.1) too, updated the code.